cortex.yaml

```diff
@@ -9,7 +9,7 @@ info:
   x-cortex-tag: go-winio
   x-cortex-type: service
   x-cortex-domain-parents:
-  - tag: endpoint
+  - tag: endpoint-mavericks
```

Comment:
Not sure why this was assigned to Mavericks.

Don't know what this repository is being used for, but going by [this pull request](https://github.com/rapid7/go-winio/pull/1) and [this pull request](https://github.com/rapid7/go-winio/pull/1/files), this might need to be assigned to the @rapid7/prism team instead.

Reply:
This repo is a agent core dependency therefore should be owned by the gophers.

Reply:
Thanks for clarifying!